### ISSUE TITLE
A11y of Configure resource pages

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2375]
There are some accessibility issues present across all themes' resource page configs:

- After marking/unmarking a block for deletion, focus is lost / returns to the beginning of the page.
- Region names, currently tagged as `<h3>`, should be `<h2>` in keeping with semantics, and the sidebar heading "Select a region" should probably have an `<h2>` tag (while keeping the `<h3>` for “Add a block to selected region” as is).
- As is certainly true for other parts of the interface, a tab pattern enabled by ARIA could be implemented for better understanding of the different tab sections (Item page, Media page, Item set page).
